packages/rest/src/lib/REST.ts

```diff
@@ -74,6 +76,16 @@ export interface RESTOptions {
 	 * @default '9'
 	 */
 	version: string;
+	/**
+	 * The amount of time in milliseconds that passes between each hash sweep
+	 * @default 3_600_000
+	 */
+	hashSweepInterval: number;
+	/**
+	 * The maximum amount of time a hash can exist in milliseconds without being hit with a request
+	 * @default 21_600_000
```

Comment:
I actually wanted some input on this from @discordjs/the-big-4, what are some reasonable defaults for the hash lifetime and the default sweep interval

Reply:
I vote 24 hours, the build up isn't too bad

Reply:
Problem with 24h intervals is that you're assuming Node.js's timers are wall clock timers, but it's not, since they're very slow.

24h intervals can run as short as a couple of hours given extreme cases. 30 minutes is generally the highest value where the timing is somewhat accurate with wall clock's time.

Reply:
I didn't realize you were talking about the interval, I was talking about the lifetime